src/test-gui/java/de/qabel/desktop/ui/accounting/AccountingGuiTest.java

```diff
@@ -32,7 +32,7 @@ public void testAddsIdentity() throws EntityNotFoundExcepion, PersistenceExcepti
         page.add().inputAndConfirm("a new identity");
 
         Identities identities = identityRepository.findAll();
-        assertEquals(1, identities.getIdentities().size());
+        assertEquals(2, identities.getIdentities().size());
```

Comment:
One is bootstrapped for all tests... because the more functionality we have, the more functionality we have that needs a persisted identity for foreign key relations (and most features need one). This seemed easier than introducing a clear method for the identities ... but it makes the tests harder to reed, indeed. //TODO